bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c

```diff
@@ -42,6 +42,26 @@ bsg_unwinder bsg_configured_unwind_style() {
   return BSG_CUSTOM_UNWIND;
 }
 
+void bugsnag_add_on_error_env(JNIEnv *env, on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = on_error;
+  }
+}
+
+void bugsnag_remove_on_error_env(JNIEnv *env, on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = NULL;
+  }
+}
+
+bool bsg_run_on_error_cbs(bsg_environment *env) {
```

Comment:
I'm not sure about using `cb`/`cbs` abbreviation - it's not immediately clear to me what it stands for. It's also not consistently applied - only used for this run method. Consider using `_callback` and/or applying to other declarations outside the public interface.

Reply:
Renamed to `bsg_run_on_error`